Makefile

```diff
@@ -14,7 +14,7 @@ SOURCES= src/lua_interpreter.cc \
 	src/qdatetime.cc \
 	src/timers.cc
 
-all: interp modules/libxml.so
+all: interp modules/libxml.so modules/libluaopenssl.so
```

Comment:
@aderiabin General point for the whole PR is to rebase this branch on top of develop